build.gradle

```diff
@@ -114,7 +119,13 @@ ext.libraries = [
     xerces: 'xerces:xercesImpl:2.9.1',
     zkclient: 'com.101tec:zkclient:0.7', // For Kafka AdminUtils
     zookeeper: 'org.apache.zookeeper:zookeeper:3.5.9',
-    zstd: 'com.github.luben:zstd-jni:1.5.2-3'
+    zstd: 'com.github.luben:zstd-jni:1.5.2-3',
+
+    grpcNettyShaded: 'io.grpc:grpc-netty-shaded:' + grpcVersion,
+    grpcProtobuf: 'io.grpc:grpc-protobuf:' + grpcVersion,
+    grpcStub: 'io.grpc:grpc-stub:' + grpcVersion,
+    tomcatAnnotations: 'org.apache.tomcat:annotations-api:6.0.53',
```

Comment:
is this dependency needed? Seems nothing related to gPRC. 😕

Reply:
To my understanding it should be necessary to include this as a `compileOnly` dependency to obtain annotations for generated code. For reference https://github.com/grpc/grpc-java/issues/9179.

Reply:
Got it.